docs/guides/service-discovery.md

```diff
@@ -0,0 +1,55 @@
+# Service Discovery
+
+## Load Balanced
+
+The load balancer hostname for a particular service can be found using `convox services`.
+
+Connecting to this hostname will distribute traffic across all processes of a given service.
+
+For an app named `myapp` with a `convox.yml` like this:
+
+    services:
+      auth:
+        port: 5000
+      web:
+        port: 3000
+
+You would see a `convox services` output similar to this:
+
+    $ convox services
+    SERVICE  DOMAIN                                PORTS
+    auth     auth.myapp.0a1b2c3d4e5f.convox.cloud  443:5000
+    web      web.myapp.0a1b2c3d4e5f.convox.cloud   443:3000
+
+The `web` service could reach the `auth` service using `https://auth.myapp.0a1b2c3d4e5f.convox.cloud:443`
+
+> Note that both of these services are available to the public internet.
+
+### Internal Services
```

Comment:
Should be only `##` rather than `###` ?

Reply:
No, this is a sub-topic of the previous topic.